docs/core/getting-started/install/win-vs2017.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install for Windows - Visual Studio 2017
+description: Install for Windows - Visual Studio 2017
```

Comment:
This should include .NET Core.

Also, should it include RC in the Visual Studio 2017 name?